src/App.js

```diff
@@ -3,28 +3,81 @@ import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+                        <li className="nav-item">
+                            <Link to="/" className="nav-link">Home</Link>
                         </li>
-                        <li>
-                            <Link to="/central-limit-theorem">Central Limit Theorem</Link>
+                        <li className="nav-item">
+                            <Link to="/central-limit-theorem" className="nav-link">Central Limit Theorem</Link>
                         </li>
                     </ul>
-                </nav>
-            </div>
-            <div className="container">
-                <Route exact path="/" component={Home} />
-                <Route path="/central-limit-theorem" component={CentralLimitGraph} />
             </div>
+            <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"> <span className="navbar-toggler-icon"></span> </button>
+        <div className="collapse navbar-collapse" id="navbarSupportedContent">
         </div>
-    </Router>
+    </nav>
+)
+
+const PreviewData = [
+        {
+            "slug": "/central-limit-theorem",
+            "title": "Central Limit Theorem",
+            "text": "Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui."
+        },
+        {
+            "slug": "/ols-regression",
+            "title": "OLS Regression",
+            "text": "Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui."
+        },
+]
+
+const PreviewBlock = (props) => (
+    <div className="col-md-4">
+        <h2>
+            <Link to={props.slug}>{ props.title }</Link>
+        </h2>
+        <p>{props.text}</p>
+    </div>
+)
+
+const Preview = () => (
+    PreviewData.map(PreviewBlock)
 )
 
 const Home = () => (
-    <h1>Home!</h1>
+    <>
+    <div className="jumbotron">
+        <div className="container">
+            <h1 className="display-3">
+                Theoremicious
+            </h1>
+            <p className="lead">
+                Seven Interactive Simulations of Statistics Theorems
+            </p>
+
+            <p>
+                Specifically designed for Doru Cojoc's Quantitative Analysis I course at SIPA, these simulations are intended as a test of effectiveness at improving learning outcomes. This project seeks to address this by turning them into web interactives, openly available. These interactives will be provided to students for their own study and experimentation, which can be used here or embedded/linked elsewhere.</p>
```

Comment:
It's best to use the `.jsx` extension for all files that use JSX - that way github and text editors can know how to do syntax highlighting based on the filename.

Reply:
👍 thanks! I didn't know that